docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Solving a <s>A</s> PyMC3 model with SMC </li><li>let's define a multivariate Gaussian distribution of dimension n <strong>with two modes</strong>, the weights <strong>of each mode</strong> and the covariance matrix. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>